.circleci/config.yml

```diff
@@ -283,3 +305,11 @@ workflows:
             - four-echo
             - five-echo
             - t2smap
+      - deploy:
+          requires:
+            - merge_coverage
+          filters:
+            tags:
+              only: /[0-9]+(\.[0-9a-zA-Z]+)*/
```

Comment:
The only thing I'm not 100% sure about is the chosen regex for our tags. In NiMARE and PyMARE, we're using the simpler `/[0-9]+(\.[0-9]+)*/`, but I didn't think that would catch the alpha and rc releases we sometimes make.

Reply:
I think we'd have to decide on a naming convention for us to make a foolproof regex, so maybe we can talk about it on the call?
I tested this and it seemed pretty reasonable for anything I could think of.

Reply:
@emdupre can correct me, but I believe our current working standard is to add `a[0-9]` (for alpha releases) or `rc[0-9]` (for release candidates) on those occasions we don't have regular releases. That said, I agree that we can/should talk about it on the call.

Reply:
Yes, let's discuss today ! But in general I'd prefer alpha _or_ rc, rather than both... But I'm guilty of having used both so far, so we should definitely standardize ! Another reason to automate :)

Reply:
I've opened #570 for the vote.

Reply:
With our decision about the rc convention, could we modify this regex?